src/lib/viewers/controls/fullscreen/FullscreenToggle.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import fullscreen from '../../../Fullscreen';
+import IconFullscreenIn24 from '../icons/IconFullscreenIn24';
```

Comment:
Is there any benefit to have these icons in box-ui-elements?

Reply:
I don't think so, since they're not part of our standardized icon set. We may refresh them at some point in the future, at which point that may make more sense.